Iced/Intel/EncoderInternal/OpHandlers.cs

```diff
@@ -629,6 +635,26 @@ sealed class OpModRM_rm : Op {
 			encoder.AddRegOrMem(ref instr, operand, regLo, regHi, allowMemOp: true, allowRegOp: true);
 	}
 
+	sealed class OpRegOpCodeByte: Op {
+		readonly Register regLo;
+		readonly Register regHi;
+
+		public OpRegOpCodeByte(Register regLo, Register regHi) {
+			this.regLo = regLo;
+			this.regHi = regHi;
+		}
+
+		public override void Encode(Encoder encoder, ref Instruction instr, int operand) 
+		{
+			if (instr.GetOpKind(operand) == OpKind.Register) 
```

Comment:
You should call encoder.Verify() to verify that it's a register OpKind.

nit: I put braces on the same line as the 'if' and the method decl.